src/internal/modules/charge-information/lib/routing.js

```diff
@@ -23,6 +23,8 @@ exports.getCreateBillingAccount = (licence, licenceHolderRole, redirect) => {
   return `/invoice-accounts/create/${region.id}/${licenceHolderRole.company.id}?${qs}`;
 };
 
+exports.postReview = request => `/licences/${request.params.licenceId}/charge-information/${request.params.chargeVersionWorkflowId}/review`;
```

Comment:
It might be better to remain consistent with the other routes and use the creatUrl() method:

exports.postReview = (chargeVersionWorkflowId, licenceId) => createUrl(`${chargeVersionWorkflowId}/review`)(licenceId);

You will also need to change the action in the form to get the licenceId and cvWorkflowId form the request object to pass it in here

Reply:
Cool. Done ✅